To: vim_dev@googlegroups.com
Subject: Patch 7.3.140
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.140
Problem:    Crash when drawing the "$" at end-of-line for list mode just after
	    the window border and 'cursorline' is set.
Solution:   Don't check for 'cursorline'. (Quentin Carbonneaux)
Files:	    src/screen.c


*** ../vim-7.3.139/src/screen.c	2011-02-09 17:09:26.000000000 +0100
--- src/screen.c	2011-03-22 13:29:04.000000000 +0100
***************
*** 5099,5109 ****
  #ifdef FEAT_DIFF
  		    || filler_todo > 0
  #endif
! 		    || (wp->w_p_list && lcs_eol != NUL && p_extra != at_end_str
! #ifdef FEAT_SYN_HL
! 			   && !wp->w_p_cul
! #endif
! 			)
  		    || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))
  		)
  	{
--- 5099,5105 ----
  #ifdef FEAT_DIFF
  		    || filler_todo > 0
  #endif
! 		    || (wp->w_p_list && lcs_eol != NUL && p_extra != at_end_str)
  		    || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))
  		)
  	{
*** ../vim-7.3.139/src/version.c	2011-03-22 13:07:19.000000000 +0100
--- src/version.c	2011-03-22 13:27:26.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     140,
  /**/

-- 
Lawmakers made it obligatory for everybody to take at least one bath
each week -- on Saturday night.
		[real standing law in Vermont, United States of America]

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
